src/resources/js/main.ts

```diff
@@ -112,8 +144,8 @@ function jsDomFunction() {
     main.prepend(section);
 
     let dynamicDiv = document.getElementById('dynamic-content');
-    appendButton(dynamicDiv, 'Add List Item', 'button-add', 'button-add dynamic-list-button', () => { modifyList('dynamic-list', 'dynamic-list-li', 'add'); });
-    appendButton(dynamicDiv, 'Remove List Item', 'button-remove', 'button-remove dynamic-list-button', () => { modifyList('dynamic-list', 'dynamic-list-li', 'remove') });
+    appendButton(dynamicDiv!, 'Add List Item', 'button-add', 'button-add dynamic-list-button', () => { modifyList('dynamic-list', 'dynamic-list-li', 'add'); });
```

Comment:
This is new, what's this `!` suffix?

Reply:
I'm guessing this is also a null assertion

Reply:
Correct - same thing here.